### PR TITLE
Landing Page

### LINKED ISSUE #5: Landing Page
- Create landing page 
- List the categories of videos

### PR DESCRIPTION
**Pull request by ttahm3d**
- Created landing page
- Added data into dbs

Closes issue #5 
localhost_3000_(iPad Mini)

### CODE REVIEW

**Review by Kothalkarpriya — APPROVED** [review]
Your code is clean and understandable. You can merge the changes

File: src/pages/landing-page/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import styles from "./LandingPage.module.css";
+
+export default function () {
```
Comment (junaidshaikh-js) [inline src/pages/landing-page/Categories.jsx:5]: I think you forgot to add the name for the component or am I missing something?

**Review by junaidshaikh-js — COMMENTED** [review]
Clean code written and easy to understand. 

Good folder structure and nice use of custom hook for the document title. 

Also loved the UI, great work!

File: src/pages/landing-page/Categories.jsx (hunk 1 above)
Comment (ttahm3d) [reply src/pages/landing-page/Categories.jsx:5]: If you are exporting component by default, you need not specify any name. Webpack will take the name of the file to be name of the Component.

Comment (junaidshaikh-js) [reply src/pages/landing-page/Categories.jsx:5]: Oh! cool.

**Pull request merged**